test/kongintegration/kongupstreampolicy_test.go

```diff
@@ -43,9 +45,10 @@ func TestKongUpstreamPolicyTranslation(t *testing.T) {
 	)
 
 	testCases := []struct {
-		name             string
-		policySpec       kongv1beta1.KongUpstreamPolicySpec
-		expectedUpstream *kong.Upstream
+		name               string
+		policySpec         kongv1beta1.KongUpstreamPolicySpec
+		expectedUpstream   *kong.Upstream
+		expectedConfigSize mo.Option[int]
```

Comment:
I don't think asserting config size in this test is necessary - it can even introduce unnecessary noise.

Reply:
Yes, but on the other hand, I'm getting higher coverage for free and it's not flaky. Only when something changes  value has to be adjusted. I'm expecting lack of error so I can expect proper size too. This is my reasoning. WDYT?

Reply:
My point is that this test is not meant to verify the in-memory strategy, it just uses it to push the config to see if it's accepted. IMO, we should aim for coverage in tests that focus on a particular component. Having such an incidental coverage might be not desired as we will have to adjust these values anytime rendered config size changes and that will be a change in theoretically non-related tests which might be confusing.

Reply:
Ok, I removed this